data/preset_defaults.json

```diff
@@ -9,6 +9,7 @@
         "amenity/place_of_worship",
         "amenity/cafe",
         "amenity/restaurant",
+        "aeroway/runway",
```

Comment:
the purpose of this file is to define the default presets which are shown to new users who have no "recently selected" presets:


<img width="314" height="694" alt="image" src="https://github.com/user-attachments/assets/ff2a0b54-8913-474f-9ed6-3c0415b37054" />

So I think the changes to this file are unrelated to the goal of this PR?